woocommerce/woocommerce
# allow env default.json to be untracked in git

### All Submissions:


### Changes proposed in this Pull Request:

This PR addresses the issue of `tests/e2e/env/config/default.json` being modified by using the e2e environment. This file is included in `.gitignore` but git ignore only works on untracked files. The PR addresses this by moving the default configuration file and copying the default to `tests/e2e/env/config/default.json` if one isn't found in `tests/e2e/config/default.json`.

### How to test the changes in this Pull Request:

1. In `master` run `npx wc-e2e docker:up`
2. Run `git status`
3. Note `tests/e2e/env/config/default.json` is listed as modified
4. Run `npx wc-e2e docker:down`
5. Run `git checkout -- tests/e2e/env/config/default.json`
6. Checkout this branch
7. Verify no files are modified according to git
8. Run E2E tests
9. Verify no files are modified according to git

### Other information:


### Changelog entry

N/A
